src/pages/Artist.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom'
+import Header from '../molecules/Header'
+import { Overview } from './Overview'
+import { Related } from './Related'
+import { About } from './About'
```

Comment:
I recommend you to create an `artist` folder and move all its sections into that folder.

So the directory structure looks like this:

```
- pages/
  - artist/
    - components/
      - Header.js // move Header from "../molecules/ to here
      - Overview.js
      - Related.js
      - About.js
    - index.js // this is your "Artist.js" now
```